docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -335,7 +335,8 @@ N/A
 
 jointags()
 ----------
-Returns all specified nodes that have **any** of the tags applied to **any** of the working set of nodes.
+Returns all specified nodes that have **any** of the tags applied to **any** of the working set of nodes,
```

Comment:
Nitpicky change suggestion: "Returns the current working set **plus** all nodes that have **any** of the tags applied to **any** node in the working set."